service/model/release.go

```diff
@@ -50,6 +50,15 @@ type Release struct {
 	UpdatedAt    time.Time
 	GitTagName   string
 	GitTagURL    url.URL
+	Attachments  []ReleaseAttachment
+}
+
+type ReleaseAttachment struct {
```

Comment:
- To keep things simple, there is only `ReleaseAttachment` struct.
- If the app needs to store more types of files in the future, it would be a good opportunity to refactor and introduce a general File model as well (IMHO).
- Although `FilePath` could serve as a natural identifier, I thing it is more flexible and practical to use UUID as ID.